src/kedro_vizro_labeling/dashboards/iris/data_manager.py

```diff
@@ -7,6 +7,8 @@
 def load_kedro_datasets(env, dataset_names=None):
     from vizro.managers import data_manager
 
+    # AM question: is there a reason you find project_path like this rather than just use Pathlib and specify
+    # Path.parent or similar?
```

Comment:
Is there a way I can access the kedro project path with Path.parent?

Something like

```
    for parent in [path] + list(path.parents):
        if (parent / ".git").is_dir():
            project_path = str(parent)
```

Reply:
Interesting! Basically checking the project dir is where there's a `pyproject.toml` with a kedro section in it rather than a `.git` folder. Makes more sense!